modules/ivc_champva/app/controllers/ivc_champva/v1/pega_controller.rb

```diff
@@ -52,7 +54,16 @@ def update_data(form_uuid, file_names, status, case_id)
 
```

Comment:
Alternatively you could collapse the above 6 lines into one:

```ruby
ivc_forms.each { |form| form.update!(pega_status: status, case_id:) }
```

Reply:
Done

Reply:
Nice!